pkg/chartutil/create.go

```diff
@@ -59,6 +61,18 @@ service:
   type: ClusterIP
   externalPort: 80
   internalPort: 80
+ingress:
+  enabled: false
+  # Used to create Ingress record (should used with service.type: ClusterIP).
+  # hostname: chart-example.local
+  annotations:
+    # kubernetes.io/ingress.class: nginx
+    # kubernetes.io/tls-acme: "true"
+  tls:
+    # Secrets must be manually created in the namespace.
+    # - secretName: chart-example.local
```

Comment:
I haven't used ingress a lot, but is there any reason this couldn't be in a secrets template and installed with Helm?